src/assets/images/dateStamp.svg

```diff
@@ -1,23 +1,103 @@
-<g filter="url(#filter0_d)">
-<path d="M376.935 83.48L441.965 53.353L451.888 36.3778L376.935 83.48Z" fill="white"/>
-<path d="M441.535 52.88L450.107 38.2171L385.364 78.9033L441.535 52.88Z" stroke="#00D0FE" stroke-width="1.21907"/>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
Because of some of the changes you made to this svg, `dateStamp.svg` grew in size. The following change reduces it down back to a reasonable size.
```suggestion
<svg width="462" height="202" version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```